package.json

```diff
@@ -103,14 +103,14 @@
 		"grunt-exec": "3.0.0",
 		"http-server": "0.11.1",
 		"husky": "1.3.1",
-		"istanbul": "0.4.5",
 		"istanbul-middleware": "0.2.2",
```

Comment:
should we don something about this `instanbul-middleware` too?

Reply:
I wanted to get rid of it too, but seems like it is being used for functional test coverage. We have to touch how we instantiate the application to remove istanbul-middleware, which I think should be done in a separate issue.

Or maybe we won't have to touch it once we write the new tests for new structure.

Reply:
@shuse2 created https://github.com/LiskHQ/lisk/issues/2919